application/templates/deployment.yaml

```
@@ -92,6 +92,10 @@ spec:
       {{- if .Values.deployment.affinity }}
       affinity:
 {{ toYaml .Values.deployment.affinity | indent 8 -}}
+      {{- end }}
+      {{- if .Values.deployment.topologySpreadConstraints }}
+      topologySpreadConstraints:
+{{ toYaml .Values.deployment.topologySpreadConstraints | indent 8 }}
       {{- end }}
```

It would be nice to try and match the enclosing indentation. Purely for readability purposes.

```suggestion
      {{- with .Values.deployment.topologySpreadConstraints }}
      topologySpreadConstraints:
        {{- toYaml . | nindent 8 }}
      {{- end }}
```